api/server/volume.go

```diff
@@ -974,12 +974,7 @@ func (vd *volAPI) snapGroup(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	snapRes, err = d.SnapshotGroup(snapReq.Id, snapReq.Labels)
-	if err != nil {
-		vd.sendError(vd.name, method, w, err.Error(), http.StatusBadRequest)
-		return
-	}
-
+	snapRes, _ = d.SnapshotGroup(snapReq.Id, snapReq.Labels)
```

Comment:
expecting drivers to not return error seems odd. @lpabon what is convention followed generally ?

Reply:
AFAIK, the interface should return an error and leave it up to the server (REST, gRPC, etc) to decide how to interpret and return the error accordingly.

So I am a little bit confused as to why we are removing #413. I think 413 is doing the right thing. Calling `sendError()` will appropriately setup the Body with the error message, and interpreted correctly by the REST client.

There is a difference between the http client throwing an error and the server returning an error. We should be careful to identify this in the client, and, afaik, it is done correctly.

@pault84 Could you provide a more complete example on to why this PR is needed?

Reply:
That is supposed to be there in my branch it's there not sure why i don't see it in this commit.